openstack/apigw/dedicated/v2/whitelist/requests.go

```diff
@@ -0,0 +1,78 @@
+package whitelist
+
+import (
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+)
+
+// CreateOpts is the structure that used to add whitelists for endpoint service.
+type CreateOpts struct {
+	// The Dedicated instance ID.
+	InstanceId string `json:"-" required:"true"`
+	// List of whitelists to add to endpoint service.
+	Permissions []string `json:"permissions" required:"true"`
+}
```

Comment:
```suggestion
// BatchOpts is the structure that used to add or remove whitelists for endpoint service.
type BatchOpts struct {
	// The Dedicated instance ID.
	InstanceId string `json:"-" required:"true"`
	// List of whitelists to add to endpoint service.
	Permissions []string `json:"permissions" required:"true"`
}
```
This structure is not just used for `Create()` method.

Reply:
fixed.